extensions/resource-deployment/src/ui/resourceTypePickerDialog.ts

```diff
@@ -313,6 +319,21 @@ export class ResourceTypePickerDialog extends DialogBase {
 		this._toolsLoadingComponent.loading = false;
 	}
 
+	private validateToolsEula(): boolean {
+		const validationSucceeded = this._tools.every(tool => {
+			const eulaValidated = tool.validateEula();
+			if (!eulaValidated) {
+				this._dialogObject.message = {
```

Comment:
This is going to overwrite the message for every tool that fails. Not a big deal now since we only have one but might as well make it work for multiples by building up the message for each failure and then setting it at the end.